modules/nf-core/software/bwa/index/functions.nf

```diff
@@ -0,0 +1,59 @@
+/*

```

Comment:
These are the functions from the modules repo right? We should probably find a common place for them, otherwise we will have them x-times in the end (if that works)

Reply:
Actually just used `nf-core modules install . bwa/index`
Maybe it's something that need to be fixed within the nf-core/modules repo

Reply:
ok, maybe this can be discussed in the modules channel. Could be tricky to figure out during the installation whether  another module has been installed and thus the functions are there.

Reply:
If it's always the same functions, then it can be in a specific path within the modules folder.
Should be easy to check if it exists

Reply:
I guess for now, here we can leave it as is